agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2406,25 +2379,47 @@ bool slave_thread::handle_cmdu_ap_manager_message(
             notification_out->mac() = notification_in->params().mac;
             message_com::send_cmdu(monitor_socket, cmdu_tx);
         }
+
         //notify master
-        // if the master is not connected, remove element from pending_client_association
         if (master_socket) {
-            auto notification_out = message_com::create_vs_message<
-                beerocks_message::cACTION_CONTROL_CLIENT_DISCONNECTED_NOTIFICATION>(
-                cmdu_tx, beerocks_header->id());
+            LOG(DEBUG) << "Controller is not connected";
+            return true;
```

Comment:
Oops! Yeah I agree that it should have been done in a separate commit but I did all the changes, and in the end start breaking it into commits, and since, and if I have broken it into separate commit, I would have needed to remove some of my changes, commit, and then add them back. It was too much headache so I decided to do it that way.